src/Dialect/ONNX/Rewrite.td

```diff
@@ -254,7 +254,7 @@ def MulAddToGemmOptPattern : Pat<(ONNXAddOp (ONNXMatMulOp:$res $m1, $m2), $m3),
 
 // onnx.add(onnx.Gemm(%X, %Y, None), %Z) = onnx.Gemm(%X, %Y, %Z)
```

Comment:
I suggest that you make beta=1.0 clear here, e.g. by writing it as
```
// onnx.add(onnx.Gemm(%X, %Y, None), %Z) = onnx.Gemm(%X, %Y, %Z) {beta=1.0}
```

Reply:
Good point! Thanks for the review